Sources/XCMetricsBackendLib/Common/Utils/Date+Utils.swift

```diff
@@ -36,6 +36,15 @@ extension Date {
         return Self.xcm_getPartitionedTableFormatter().string(from: self)
     }
 
+    /// Sets the time components of a date to 0, with no regards to time zone
+    /// which can be useful when comparing to database dates (with no time information)
+    /// - Returns: A `Date` with time components at 0
+    func truncateTime() -> Date {
```

Comment:
Can you add the prefix xcm? so it will be `xcm_truncateTime`? It's a convention used to highlight that it's a function created by us in a Type that's is part of Swift.